### PR TITLE
Update CI to run verify on zone & exchange changes

### PR DESCRIPTION
**Pull request by VIKTORVAV99**
The changes should be pretty self explanatory but the gist of it is that this PR makes our CI more robust by checking if `excluded-aggregate-exchanges.json` should update or not (matching `world.json` behavior) and changes the verify triggers to run on `/geo/`, `/config/zones/` and `/config/exchanges/` changes.

Also I added the updated `excluded-aggregate-exchanges.json` I forgot to check in #4800.

### CODE REVIEW

File: .github/workflows/CI.yml [hunk 1]
```diff
@@ -99,7 +111,7 @@ jobs:
   verify:
     name: Verify
     needs: [setup, prettier]
-    if: needs.setup.outputs.frontend != 'false' || needs.setup.outputs.workflows != 'false'
+    if: needs.setup.outputs.geometries != 'false' || needs.setup.outputs.workflows != 'false'
```
Comment (madsnedergaard) [inline .github/workflows/CI.yml:114]: Is this change on purpose - shouldn't it still run on frontend changes? 🤔

**Review by madsnedergaard — APPROVED** [review]
One minor question, otherwise LGTM!

File: .github/workflows/CI.yml (hunk 1 above)
Comment (VIKTORVAV99) [reply .github/workflows/CI.yml:114]: It now runs on all the web/geo changes and zone + exchange changes. There should be no need to check if the map has updated if we are just changing "pure" frontend components like say the translations.

Comment (VIKTORVAV99) [reply .github/workflows/CI.yml:114]: @madsnedergaard merging this now, but if there are any issues, we should spot them pretty easily as we run this and the CirlceCI jobs in parallel.

Comment (madsnedergaard) [reply .github/workflows/CI.yml:114]: Ah okay, I thought we would need either `.frontend` or `.geometries` to be not false, but makes sense :)

Comment (VIKTORVAV99) [reply .github/workflows/CI.yml:114]: Hmm, good point. It might be needed for cypress since it needs verify. I'll monitor the situation and add `.frontend` too if it's needed.
